assets/sass/_base.scss

```diff
@@ -1,20 +1,3 @@
-html {
```

Comment:
Can you try importing this into okta-core and making sure that this does not change the styling of the login page?